src/pages/MapPage/components/Map/Map.js

```diff
@@ -155,29 +161,62 @@ function Map(props) {
                       lat: parseFloat(tag.coordinates.latitude),
                       lng: parseFloat(tag.coordinates.longitude)
                     }}
-                    icon={
-                      activeTagId === tag.id
-                        ? {
-                            url:
-                              missionredImage[
-                                missionName.findIndex(
-                                  (mission) =>
-                                    mission === tag.category.missionName
-                                )
-                              ],
+                    icon={(() => {
+                      if (activeTagId === tag.id) {
+                        return {
+                          url:
+                            missionredImage[
+                              missionName.findIndex(
+                                (mission) =>
+                                  mission === tag.category.missionName
+                              )
+                            ],
+                          scaledSize: { width: 20, height: 20 }
+                        }
+                      }
+                      if (tag.category.missionName === '動態任務') {
+                        if (
+                          compareTime(tag.lastUpdateTime) ===
+                            'a few seconds ago' ||
+                          compareTime(tag.lastUpdateTime)[2] === 'm'
+                        ) {
+                          return {
+                            url: missionImage[3],
                             scaledSize: { width: 20, height: 20 }
                           }
-                        : {
-                            url:
-                              missionImage[
-                                missionName.findIndex(
-                                  (mission) =>
-                                    mission === tag.category.missionName
-                                )
-                              ],
+                        }
+                        if (
+                          compareTime(tag.lastUpdateTime)[0] === 'a' &&
+                          compareTime(tag.lastUpdateTime)[2] === 'm'
+                        ) {
+                          return {
+                            url: missionImage[3],
                             scaledSize: { width: 20, height: 20 }
                           }
-                    }
+                        }
+                        if (compareTime(tag.lastUpdateTime)[3] === 'm') {
+                          if (compareTime(tag.lastUpdateTime)[0] < '3') {
+                            return {
+                              url: missionImage[3],
+                              scaledSize: { width: 20, height: 20 }
+                            }
+                          }
```

Comment:
You can use comparison function like this to check if it's beyond 30 minutes. You can check it on mementjs's official documant. Please do not check with string.
<img width="538" alt="Screen Shot 2021-07-06 at 6 18 37 PM" src="https://user-images.githubusercontent.com/48475056/124584480-e1c99300-de86-11eb-869b-8e413cafe605.png">

Reply:
Already change follow above code